config/config.go

```diff
@@ -266,6 +266,22 @@ func (c *C) GetBool(k string, d bool) bool {
 	return v
 }
 
+func AsBool(v any) (value bool, ok bool) {
+	switch x := v.(type) {
+	case bool:
+		return x, true
+	case string:
+		switch x {
```

Comment:
Does this work for `Yes`?

Reply:
ah this is what they are doing in v3 if you unmarshal into a Bool:

- https://github.com/go-yaml/yaml/commit/01167db92b45abedd6aa8435e46d1e75097dcc4b

So we can probably copy that.

This is what v2 did:

- https://github.com/go-yaml/yaml/blob/v2.4.0/resolve.go#L37-L42

Reply:
Existing in v1.9.5: https://github.com/slackhq/nebula/blob/v1.9.5/config/config.go#L257-L262